# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1362019529 18000
#      Wed Feb 27 21:45:29 2013 -0500
# Node ID aa5e1e8dce6676387964bc7c108c0e8ebbe5c55c
# Parent  a57c2c8c816365ed4c0fc50e0e8cb6b5179ead5d
* lex.ll: Change y.tab.h to oct-parse.h in comment.

diff --git a/libinterp/parse-tree/lex.ll b/libinterp/parse-tree/lex.ll
--- a/libinterp/parse-tree/lex.ll
+++ b/libinterp/parse-tree/lex.ll
@@ -62,19 +62,19 @@ object) relevant global values before an
 
 #include <sys/types.h>
 #include <unistd.h>
 
 #include "cmd-edit.h"
 #include "quit.h"
 #include "lo-mappers.h"
 
-// These would be alphabetical, but y.tab.h must be included before
-// oct-gperf.h and y.tab.h must be included after token.h and the tree
-// class declarations.  We can't include y.tab.h in oct-gperf.h
+// These would be alphabetical, but oct-parse.h must be included before
+// oct-gperf.h and oct-parse.h must be included after token.h and the tree
+// class declarations.  We can't include oct-parse.h in oct-gperf.h
 // because it may not be protected to allow it to be included multiple
 // times.
 
 #include "Cell.h"
 #include "comment-list.h"
 #include "defun.h"
 #include "error.h"
 #include "gripes.h"
